packages/driver/src/cy/commands/sessions/manager.ts

```diff
@@ -227,7 +238,7 @@ export default class SessionsManager {
       *      send a message telling our proxy server to intercept the next req to the https domain,
       *      then follow 2)
       */
-    getStorage: async (options = {}) => {
```

Comment:
Now that we are exposing these as commands,  do we need to include the global session API? I don't think we need to since they are currently undocumented.

Reply:
Not sure I understand. What would including the global session API mean?